website/docs/language/backend/s3.mdx

```diff
@@ -62,25 +84,40 @@ This is seen in the following AWS IAM Statement:
       "Effect": "Allow",
       "Action": "s3:ListBucket",
       "Resource": "arn:aws:s3:::mybucket"
```

Comment:
```suggestion
      "Resource": "arn:aws:s3:::mybucket",
```

Missing comma, [which also caused a syntax highlighter issue.](https://terraform-c0hy2rx5y-hashicorp.vercel.app/terraform/language/backend/s3#s3-bucket-permissions:~:text=not%20delete%20it.-,This%20is%20seen%20in%20the%20following%20AWS%20IAM%20Statement%3A,-%7B)

![Screenshot 2025-01-07 at 1 09 31 PM](https://github.com/user-attachments/assets/8144e76a-6169-4351-8139-7a15041ed1ed)

Reply:
Thanks Ruben, fixed!